.npmrc

```diff
@@ -1,18 +1,3 @@
 shamefully-hoist=true  
 
-# TODO: hopefully can remove after migration from nuxt/composition-api to nuxt bridge
-
-# (waiting on response from: https://github.com/nuxt-community/composition-api/issues/573)
```

Comment:
Why do we not need this explanation anymore?

Reply:
I assumed it was because we skipped Nuxt Bridge altogether and went straight into Nuxt 3. Could be wrong though

Reply:
Ya i just changed it because we're on Nuxt 3, i tried disabling it but it's still causing issues. I'll figure this out later.